angular2/angular2.d.ts

```diff
@@ -12,611 +12,701 @@ interface List<T> extends Array<T> {}
 interface Type {}
 
 interface _ComponentArg {
-  /**
-   * The CSS selector that triggers the instantiation of a directive.
-   *
-   * Angular only allows directives to trigger on CSS selectors that do not cross element boundaries.
-   *
-   * `selector` may be declared as one of the following:
-   *
-   * - `element-name`: select by element name.
-   * - `.class`: select by class name.
-   * - `[attribute]`: select by attribute name.
-   * - `[attribute=value]`: select by attribute name and value.
-   * - `:not(sub_selector)`: select only if the element does not match the `sub_selector`.
-   * - `selector1, selector2`: select if either `selector1` or `selector2` matches.
-   *
-   *
-   * ## Example
-   *
-   * Suppose we have a directive with an `input[type=text]` selector.
-   *
-   * And the following HTML:
-   *
-   * ```html
-   * <form>
-   *   <input type="text">
-   *   <input type="radio">
-   * <form>
-   * ```
-   *
-   * The directive would only be instantiated on the `<input type="text">` element.
-   *
-   */
   selector: string;
```

Comment:
Why did you remove the documentation? It's useful in an editor where it's shown side-by-side with the auto-completion.